Cargo.toml

```diff
@@ -65,9 +65,7 @@ syn = { version = "2", features = ["full", "extra-traits", "visit-mut"] }
 strum = { version = "0.26.3", features = ["derive"] }
 tracel-xtask = { version = "~1.0" }
 
-portable-atomic-util = { version = "0.2.2", features = [
-    "alloc",
-] } # alloc is for no_std
+portable-atomic = { version = "1.9" }
```

Comment:
Don't think you need this at all anymore?

Reply:
We kind of do, because we're activating the `portable-atomic` feature of `spin` and it fails the burn CI if we don't activate the right feature flag on `portable-atomic` when native atomic aren't available... yeah I know it's painful 😅